Makefile

```diff
@@ -216,6 +220,7 @@ install-crds:
 	kubectl apply -f https://raw.githubusercontent.com/stolostron/governance-policy-propagator/main/deploy/crds/policy.open-cluster-management.io_policies.yaml
 	kubectl apply -f deploy/crds/policy.open-cluster-management.io_configurationpolicies.yaml
 	kubectl apply -f deploy/crds/policy.open-cluster-management.io_operatorpolicies.yaml
+	kubectl apply -f https://raw.githubusercontent.com/operator-framework/api/master/crds/operators.coreos.com_olmconfigs.yaml
```

Comment:
I'm not sure why this is necessary. Isn't this installed with `install.sh` above?

Reply:
It does look like `install.sh` handles the CRDs. I'll try it without this.

Reply:
I needed a test PR, so I updated it here:
- https://github.com/open-cluster-management-io/config-policy-controller/pull/190